utils/iscsi-pr.c

```diff
@@ -0,0 +1,491 @@
+/* 
+   Copyright (C) 2022 by zhenwei pi<pizhenwei@bytedance.com>
+
+   This program is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation; either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifdef HAVE_CONFIG_H
```

Comment:
Can this code block be left out? I don't see any poll() calls so why has <poll.h> been included? And if #include <poll.h> is left out, why to include <config.h>?

Reply:
Yes, removed this block and other 2 header files, and force pushed a new version.
Thanks a lot!